axis_backend/apps/kpis/views/kpi_type_viewset.py

```
@@ -0,0 +1,63 @@
+    Provides standard CRUD endpoints for KPI types.
+    """
+
+    queryset = KPIType.objects.all()
+    permission_classes = [IsAuthenticated]
+    service_class = KPITypeService
+    list_serializer_class = KPITypeListSerializer
+    detail_serializer_class = KPITypeDetailSerializer
+    create_serializer_class = KPITypeCreateSerializer
+    update_serializer_class = KPITypeUpdateSerializer
+
+    def create(self, request, *args, **kwargs):
```

Ruff (0.14.5)

Missing type annotation for `*args`

(ANN002)